sdks/java/io/cdap/src/test/java/org/apache/beam/sdk/io/cdap/CdapIOTest.java

```diff
@@ -118,12 +118,6 @@ public void testReadObjectCreationFailsIfValueClassIsNull() {
         IllegalArgumentException.class, () -> CdapIO.<String, String>read().withValueClass(null));
   }
 
-  @Test
-  public void testReadValidationFailsMissingCdapPluginClass() {
```

Comment:
Good to preserve these error handling tests. now it throws at expansion.

Reply:
Done